scripts/tests/create_project_table_test.py

```diff
@@ -0,0 +1,139 @@
+"""
+Unit tests create_project_table.py.
+
+This suite tests functions for extracting metadata, generating markdown table rows, 
+processing multiple README files, and inserting rows into tables, using mocks to simulate file operations.
+"""
+
+import unittest
+from unittest.mock import MagicMock, patch
+from pathlib import Path
+import sys
+
+sys.path.append(str(Path(__file__).resolve().parent.parent))
+
+from create_project_table import (
```

Comment:
- We do not import functions, we import modules (some exceptions apply, even in this PR, but not here)
- We do not group imports in a block, this is Python, not Go